downstairs/src/region.rs

```diff
@@ -605,11 +649,46 @@ impl Region {
             }
         };
         save_stream_to_file(extent_copy, repair_stream.into_inner()).await?;
+        let mut count = 1;
+
+        // If we are replacing our region with one from an older version
+        // that contained the SQLite, then look for and copy the database
+        // files.
+        for opt_file in &[ExtentType::Db, ExtentType::DbShm, ExtentType::DbWal]
+        {
+            let filename = extent_file_name(eid as u32, opt_file.clone());
```

Comment:
Nit: `ExtentType` should derive `Copy`, letting you remove this clone:
```suggestion
            let filename = extent_file_name(eid as u32, opt_file);
```

Reply:
Done in [7bef555](https://github.com/oxidecomputer/crucible/commit/7bef55560bd1a345855ca77b6c296616ee66e0fe)